src/core/ack_tracker.c

```diff
@@ -96,6 +96,39 @@ QuicAckTrackerAddPacketNumber(
         !RangeUpdated;
 }
 
+BOOLEAN
+QuicAckTrackerDidHitReorderingThreshold(
+    _In_ QUIC_ACK_TRACKER* Tracker,
+    _In_ uint8_t ReorderingThreshold
+    )
+{
+    if (ReorderingThreshold == 0) {
+        return FALSE;
+    }
+
+    uint64_t LargestUnackedPacketNumber = QuicRangeGetMax(&Tracker->PacketNumbersToAck);
```

Comment:
Let's name these exactly what the spec calls them.
```suggestion
    uint64_t LargestUnacked = QuicRangeGetMax(&Tracker->PacketNumbersToAck);
```